### ISSUE TITLE
p and q parameters of node2vec

### ISSUE THREAD

**Issue by Tixierae** [palash1992/GEM#1]
Hi Palash,

I have a question about your paper. I am aware this may not be the best place to ask, but I didn't know where else to post - hope you don't mind.

Maybe I missed it, but in your experiments you don't seem to report the values of the parameters `p` and `q` that you used for node2vec. Did you optimize those values, for each dataset? If yes, how? Note that if you used the default settings `p=q=1`, then you were using DeepWalk, not node2vec.

BTW, in Table 1, you indicate the wrong reference for DeepWalk - it is not S. Skienna et al. but Perozzi et al. And, just out of curiosity, still in Table 1, how did you come up with the time complexity values for DeepWalk and node2vec? I couldn't find those estimates in the original papers.

Thanks!

**Comment by palash1992** [comment]
Hi Antoine,

Thanks for your feedback. I manually varied the values of p and q. For SBM,
I kept p=1 and q=256 but for most of the other data sets the performance
didn't vary much with change in p and q (maybe by 0.1) so I didn't vary
them much. I know further optimization may improve the results a bit but
even in the original paper they had performed grid search on a small set of
parameters.

Thanks for pointing out the mistake in the table. I will correct it. I
calculated the time complexity myself on the basis of the time analysis
paragraph in node2vec.

Regards,
Palash Goyal
